conf/ftp

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+# Copying cert.pem and dhparams.pem to pure-ftpd.pem and pure-ftpd-dhparams.pem.
+cp /etc/ssl/private/cert.pem /etc/ssl/private/pure-ftpd.pem
```

Comment:
Rather than copy the default cert, I'd be inclined to either reconfigure pure-ftpd to use the default (i.e. `/etc/ssl/private/cert.pem`) or, if that's not easily done, just create symlinks (rather than copies).

My rationale for that is that by default these certs should be (re)generated at first boot, but the default certs are already regenerated. So if you leverage the defaults (either via using them directly or via symlinks) then you get their regeneration for free.

Also, it means that if a user configures Let's Encrypt, then FTPS will use the proper TLS/SSL certs by default too (and again for free).

Reply:
I have reconfigured pure-ftpd to use the default cert located at `/etc/ssl/private/cert.pem`, but to do that i had to change the mechanism to use the `.conf` file instead, for which i had to make the following changes in `\etc\init.d\pure-ftpd` script.
- Disabled the `pure-ftpd-wrapper` which reads the config parameters files created in the `/etc/pure-ftpd/conf/` folder, for e.g. `/etc/pure-ftpd/conf/AllowUserFXP` and converts them into parameters being passed into pure-ftpd executable.
- Enabled the pure-ftpd executable to read the parameters from `.conf` file instead (On `start` and `restart` actions).

I am ready with the changes, and can update repo/PR, if you think its fine.

Reply:
> to do that i had to change the mechanism to use the .conf file instead, for which i had to make the following changes in \etc\init.d\pure-ftpd script.

Hmm, did you test using a symlink? Perhaps that might "just work"?

Also, by default the cert.pem and cert.key files are only readable by root, so be aware of that.

Reply:
I am sure symlink would work out of the box, I just wanted to use `.conf` file method instead. 

Regd the readable by root, I already have implemented and tested the `.conf` file which is using the default cert file and it has no problem reading it.

Reply:
@JedMeister 

- I have tested using the `linking method`, creating symbolic link of cert.pem and dhparams.pem, it is working without any issues.
- I have also tested using the `.conf file method`, that is also working fine.

Please do suggest which method do you want me to implement?

Reply:
@JedMeister Reply awaited.

Reply:
Thanks for the bump @anandphulwani - apologies on my slow response...

Let's just go with the symlink.

Reply:
@JedMeister Have pushed the changes.